metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/utils/MongoDatabaseUtils.java

```diff
@@ -41,10 +41,14 @@
  *
  * @author Yorgos.Mamakis@ kb.nl
  */
-@SuppressWarnings("rawtypes")
 public class MongoDatabaseUtils<T> {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MongoDatabaseUtils.class);
+  
+  private static final String AGENT_TABLE = "people";
+  private static final String CONCEPT_TABLE = "concept";
+  private static final String PLACE_TABLE = "place";
+  private static final String TIMESPAN_TABLE = "period";
 
   private static JacksonDBCollection<ConceptTermList, String> cColl;
   private static JacksonDBCollection<PlaceTermList, String> pColl;
```

Comment:
It's inside the method but I cannot comment. It's about the Mongo class used. Could you upgrade the version on mongo and use it the same way as in the Application.java in the metis-core-rest?

Reply:
There is also a syserr inside, which can be replace by a logger.

Reply:
And on dbExists add synchronized modifier.

Reply:
I think the upgrade of MongoJack (which is needed... see TODO in the code) must wait for later maintenance.

Fixed the other two issues.